metrics/prometheus/kubernetes.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: prometheus
+    spec:
+      containers:
+      - name: prometheus
+        image: prom/prometheus:v2.0.0
+        imagePullPolicy: IfNotPresent
+        ports:
+        - name: prometheus
```

Comment:
The Prometheus port is called `prometheus` ... will it scrape it self for the metrics?

Reply:
Yes it's scraping for its metrics, maybe changing our metrics pods ports is better than changing this one. What do you think ?

Reply:
Yeah, we can change the names on our pods instead. Using something such as `metrics` should be fine.